checkout-ui-custom/src/partials/Deliver/utils.js

```diff
@@ -398,6 +399,15 @@ export const setAddress = (address, options = { validateExtraFields: true }) =>
     .done(() => clearLoaders());
 };
 
+export const showAlertBox = () => {
+  $('.alert-container').addClass('show');
+  $('#bash-alert-container').html(Alert({ text: 'Address added' }));
+  // After 5 seconds, remove the element
+  setTimeout(() => {
+    $('.alert-container').remove();
```

Comment:
Here we could use jQuery .slideUp() to slide up the element, then remove it afterwards. Not essential - nice to have.